Cloud Application Manager/Core Concepts/architecture-components.md

```diff
@@ -0,0 +1,77 @@
+{{{
+"title": "Architecture components",
+"date": "05-22-2019",
+"author": "Diego Sanjuan",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [CAM Agents](#cam-agents)
+* [Websockets](#websockets)
+* [API Services](#api-services)
+* [Orchestration Workflow](#orchestration-workflow)
+* [Connectivity and Required Firewall Rules](#connectivity-and-required-firewall-rules)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+
+### Overview
+
+This article is meant to assist users of Cloud Application Manager to learn about its components and interaction as follows:
+
+* CAM agents - functions, interactions, installation, and requirements
+* Websockets - functions, interactions
+* API Services - functions, interactions
+* Orchestration Workflow - functions, interactions
+* Connectivity and Required Firewall Rules
+
+
+### Audience
+
+All Cloud Application Manager users who want to learn about Cloud Application Manager component's architecture and its connectivity requirements.
+
+
+### CAM Agents
+Are responsible of execute event scripts in virtual machines deployed using script boxes and return their execution results. Also provide to backend network information about virtual machine regularly to inform user about virtual machine availability.
+Agent is bootstrapped and installed, in each virtual machine registered in CAM, making use of diverse execution mechanisms provided by public and private cloud providers. i.e. in AWS cloud init is used for download and install it.
```

Comment:
Please, include a blank line to separate paragraphs

Reply:
done